docs/completeExamples/completeExamples.xml

```diff
@@ -0,0 +1,485 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<Events xmlns="event-logging:3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="event-logging:3 file://event-logging-v3.0.0.xsd" Version="3.0.0">
+    <Event>
+        <EventTime>
+            <TimeCreated>2017-01-02T03:04:05.678Z</TimeCreated>
+        </EventTime>
+        <EventSource>
+            <System>
+                <Name>FileSharer21</Name>
+                <Description>Interactive File Sharing</Description>
+                <Environment>Operational</Environment>
+                <Organisation>ACMECoolResearch</Organisation>
+            </System>
+            <Generator>fs-21-v2.2</Generator>
+            <Device>
+                <HostName>fs04.fs.myorg.com</HostName>
+                <IPAddress>131.141.151.161</IPAddress>
+                <MACAddress>A1:B1:C1:D1:E1:F1</MACAddress>
+            </Device>
+            <Client>
+                <IPAddress>121.121.121.121</IPAddress>
+            </Client>
+            <User>
+                <Id>jc101</Id>
+                <UserDetails>
+                    <Id>jc101</Id>
+                    <StaffNumber>101</StaffNumber>
+                    <Surname>Coder</Surname>
```

Comment:
are we missing first name or family name in here.

I think we could do with an exercise to go through all the elements in the schema xsd and ensure we have decent field level examples/descritpions in the annotations.  Thus making the document more self describing.  We already have a lot of annotations but I suspect there are lots of gaps.